Dependencies/napi/CMakeLists.txt

```diff
@@ -3,8 +3,11 @@ cmake_minimum_required(VERSION 3.12)
 project(napi)
 
 # Set per-platform defaults if no JS engine is set.
-if (WIN32)
+if (WINDOWS_STORE)
     set(NAPI_JAVASCRIPT_ENGINE "Chakra" CACHE STRING "JavaScript engine for N-API.")
+elseif (WIN32)
+    set(NAPI_JAVASCRIPT_ENGINE "V8" CACHE STRING "JavaScript engine for N-API.")
+    set(V8_USE_INSPECTOR ON CACHE BOOL "Enable the V8 inspector.")
```

Comment:
This will make the default V8. Is that what we want?

Reply:
I think it makes the most sense feature-wise to make V8 with inspector the default. If folks think otherwise I can remove the line.

Reply:
I don't think we should unless all platforms do the same.